### DISCUSSION TITLE
"Connection failed" and Discord doesn't prompt auth approval

### DISCUSSION THREAD

**Discussion by Oxterel** [fredemmott/StreamDeck-Discord#40] [Q&A] [answered]
Hello!

It seems like I can't get a connection to Discord.

I am using https://localhost/ as redirect URI and I am copying the client ID and client secret in to the button on Stream Deck, but discord never prompts with me with an authentication dialougue and the stream deck button just says "Connection failed" in debug state.

I've tried restarting the computer, discord and stream deck with same result. I've tried hitting the Reconnect and Reauthenticate buttons with same result. I've tried uninstalling and reinstalling the plugin with same result.

**Comment by Oxterel** [comment]
Both Discord and Stream Deck is running as admin.

**Comment by Oxterel** [comment, marked as answer]
Figured it out. Appearently it doesn't like being run in admin mode.

**Comment by SuperMiniMicro** [comment]
Did you take any other steps? I'm also getting this error, but neither program is running in admin mode.

**Reply by Oxterel** [reply]
I didn't do anything but disabling the admin mode, sorry.
